report_app/templates/results.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Search Results</h1>
+
+    <!-- Inserting a search bar here -->
+    <form action="/search-results-public" method="get" class="govuk-form-group">
+      <div class="govuk-form-group">
+        <input class="govuk-input" id="search" name="q" type="text" placeholder="Repository Name" autocomplete="off">
+      </div>
+      <button class="govuk-button" type="submit" style="margin-top: 10px;">Search</button>
+    </form>
+    <ul class="govuk-list govuk-list--bullet">
```

Comment:
does it need a closing ul element <\ul> ?

Reply:
Good catch, Nick. I've now added the closing tag.